gitreload/config.py

```diff
@@ -61,7 +62,7 @@ def configure_logging(level_override=None, config=Config):
     address = None
     if config.LOG_FILE_PATH:
         address = config.LOG_FILE_PATH
-    elif os.path.exists('/dev/log'):
+    elif Path('/dev/log').is_file():
```

Comment:
Should this be `is_dir`?

Reply:
I'm pretty sure it's intended to be a file object and not a directory.